ajax/libs/rollbar.js/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "rollbar.js",
+  "author": "Rollbar, Inc",
+  "description": "Error tracking and logging from JS to Rollbar",
+  "version": "1.9.2",
+  "homepage": "https://rollbar.com",
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/rollbar/rollbar.js"
+  },
+  "license": "MIT",
+  "filename": "rollbar.umd.nojson.min.js",
+  "dependencies": {
+    "console-polyfill": "0.2.2",
+    "error-stack-parser": "1.3.3",
+    "extend": "3.0.0"
+  },
```

Comment:
@jondeandres I think we don't need `dependencies` field in **package.json**, please remove it.